components/provisioner/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/gocraft/dbr/v2 v2.6.3
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
+	github.com/hashicorp/go-version v1.2.0
```

Comment:
This version was released 23 Apr 2019. I think we could use the most recent one 1.4.0